executor/runner/runner.go

```diff
@@ -349,6 +352,26 @@ no_launchguard:
 	}
 }
 
+func (r *Runner) handleTaskRunningMessage(ctx context.Context, msg string, lastMessage *string, runningSent *bool, startTime time.Time, details *runtimeTypes.Details) {
```

Comment:
This exists mostly to keep the cyclomatic complexity of the monitorContainer function low.